FerramAerospaceResearch/FARGUI/FAREditorGUI/Simulation/InstantConditionSim.cs

```diff
@@ -141,15 +141,15 @@ public void GetClCdCmSteady(InstantConditionSimInput input, out InstantCondition
             AngVel += (sinPhi * alphaDot - cosAlpha * cosPhi * betaDot) * up;
 
             Vector3d velocity = forward * cosAlpha * cosBeta;
-            velocity += right * (sinPhi * cosAlpha * cosBeta + cosPhi * sinBeta);
-            velocity += -up * cosPhi * (sinAlpha * cosBeta + sinBeta);
-
-            velocity.Normalize();
+            velocity += right * (sinPhi * sinAlpha * cosBeta + cosPhi * sinBeta); // Rodhern: Changed velocity 'right' and 'up' components
+            velocity += -up * (cosPhi * sinAlpha * cosBeta - sinPhi * sinBeta);   //  to align vector to naive forward direction.
```

Comment:
Which order are the rotations applied in? Need to be sure the math is correct before merging

Reply:
Are you sure you used the formulas from my code? Did you change the components back to the code from before the merge on purpose?